### PR TITLE
Gary's code smells

### PR DESCRIPTION
**Pull request by gary510**
- deleted handlebar files for categories
- created one handlebar file that will function for all categories
- updated the main.js so one  function will take care of all categories
-deleted a " BuildRecipeViewController.js " file that didnt seem to have function.

### CODE REVIEW

**Comment by anthonyliuhao** [comment]
The redundant `BuildRecipeViewController.js` file was removed in [this commit](https://github.com/anthonyliuhao/cs361_group_project/commit/202b1c81be355111f3190858dee23a434bf07b5a#diff-75f8251909b8e9c19d481378c81a6c10b7619a73d4ad5f9be73835ca08bd8ac2). Looks like you were just working off of an outdated `development` branch?

File: main.js [hunk 1]
```diff
@@ -34,70 +34,36 @@ app.get('/build',function(req,res,next){
...
-
-/*
-display recipes for dinner
-*/
-app.get('/dinner',function(req,res,next){
+app.get('/category/:catname',function(req,res,next){
   let context = {};
-  context.title = "Dinner";
+  var type = req.params.catname;
+  context.title = "Ethical Eating - " + type;
```
Comment (anthonyliuhao) [inline main.js]: `main.handlebars` already contains the `Ethical Eating - ` part of the title, so you don't need to repeat it here.

File: main.js [hunk 2]
```diff
@@ -34,70 +34,36 @@ app.get('/build',function(req,res,next){
...
-    res.render('lunch', context);
-  });
-});
-
-
-/*
-display recipes for dinner
-*/
-app.get('/dinner',function(req,res,next){
+app.get('/category/:catname',function(req,res,next){
```
Comment (anthonyliuhao) [inline main.js]: Can we simply use `/category/:category`, instead of `/category/:catname`?

**Review by anthonyliuhao — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (gary510) [reply main.js]: this is updated.

File: main.js (hunk 2 above)
Comment (gary510) [reply main.js]: I updated this for clarity.

**Comment by gary510** [comment]
@anthonyliuhao I merged the dev branch into this one and it should be up to date now.

**Review by anthonyliuhao — DISMISSED** [review]
Looks good. Thanks!

**Review by anthonyliuhao — APPROVED** [review]

**Pull request merged**